### ISSUE TITLE
Inviting existing accounts to be RAs (upgrading)

### ISSUE THREAD

**Issue by joemsak** [Iridescent-CM/technovation-app#1376]
Some RAs already have accounts on the platform, but not RA profiles

Tweak the "Invite users" system to allow for inviting existing accounts to 'upgrade' to RA status

QA

* Ensure that a mentor account exists with an email you can access, and note the email
* Login as this mentor and visit your teams / profile page - remember the details
* Login as the Admin on QA
* Go to "Invite users"
* Invite an RA with the same email as the mentor
* Use the invite email to sign up as an RA
* Go to "Mentor Mode" (top left corner of dashboard)
* Ensure that the same mentor is attached to this new RA - same teams, same profile
* This new RA will have to set their location, regardless of the previous mentor location

**Comment by joemsak** [comment]
@kmorton2017 when you are available, I will want you to review this feature. Thanks! Anyone else can review it in the meantime

**Comment by ophelieh** [comment]
Notes (the first 2 are FYI)
* I used a different password to create my RA account and it changed my previous password to the new one
* I used a different name for the mentor and RA and it picked the RA name over the mentor's
* I set myself to SF though so I have to pass the background check before I can see my RA dashboard and I cannot change my location myself since I'm an RA. 
* Also, I am pretty sure I had signed a consent waiver before signing up an RA and it is asking me to do it again. Is that normal? 

Other than that, everything looks good. :)
